src/lib/commands.js

```diff
@@ -0,0 +1,271 @@
+import { TCModel, GVL } from "@iabtcf/core";
+import config from "./config";
+import {
+	encodeVendorConsentData,
+	encodeConsentData,
+} from "./cookie/cookie";
+const arrayFrom = require('core-js/library/fn/array/from');
+
+const COOKIE_VERSION = 2;
+const CMP_ID = 280;
+const CMP_VERSION = 2;
```

Comment:
These values are already defined in "./init" module. Should be configured in a single place.

Reply:
done